approved/v93k/prb1.tf

```diff
@@ -1838,11 +1810,11 @@ test_flow
         {
         }
         run(erase_all_40_864CE8F);
-      }
-      else
-      {
-      }
-    }, open,"additional_erase", ""
+      }, open,"additional_erase", ""
+    }
+    else
+    {
+    }
```

Comment:
Logically the same, just that the job context is now the outer one.
The new output aligns to the order the conditions are applied in the source file.